cmd/rizzod/main.go

```diff
@@ -293,7 +293,7 @@ func lockComment() (string, error) {
 }
 
 func parseStderr(oid string, stderr string) (*rizzopb.PuppetReport, error) {
-	var regexEvalErr = regexp.MustCompile(`^Error: Evaluation Error:`)
+	var regexEvalErr = regexp.MustCompile(`^Error: (?:Evaluation E|Syntax e)rror`)
```

Comment:
I think this is slightly easier to read, although it does cover up the fact that there is a disparity between the two errors' capitalization standards. Your call whether you think this is a good change or not.

```suggestion
	var regexEvalErr = regexp.MustCompile(`^Error: (?:Evaluation|Syntax) [Ee]rror`)
```

Reply:
Yeah I think that's probably clearer... I guess in the end I'm indifferent about it, but it seems reasonable